fabric-object-builder-api-v1/src/testmod/java/net/fabricmc/fabric/test/object/builder/ObjectBuilderTestConstants.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.test.object.builder;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import net.minecraft.util.Identifier;
+
+public final class ObjectBuilderTestConstants {
+	public static final String MOD_ID = "fabric-object-builder-api-v1-testmod";
+	public static final Logger LOGGER = LogManager.getLogger(MOD_ID);
+
+	public static Identifier id(String name) {
```

Comment:
not needed IMO, it used once as well.

Reply:
will be used by other test entrypoints in the future

Reply:
I'd say add it when needed

Reply:
This just makes code much cleaner and way more maintainable when you updates the mod id

Reply:
> 
> 
> I'd say add it when needed

Just like "I'd say add this constant class when needed", this will cause extra refactors if two prs add test entrypoints simutaneously after this one is merged